omnifold/omnifold.py

```diff
@@ -194,7 +198,10 @@ def RunStep2(self,i):
         )
         new_weights = np.ones_like(self.weights_push)
         new_weights[self.mc.pass_gen]=self.reweight(self.mc.gen,self.model2)[self.mc.pass_gen]
+        # new_weights[self.mc.pass_gen]=self.reweight(self.mc.gen,self.step2_models[0])[self.mc.pass_gen]
+        print(self.weights_push)
         self.weights_push = new_weights
+        print(self.weights_push)
```

Comment:
same here

Reply:
Done. Also took out the print's in CompielModels.